data_visualization_in_seaborn/data_visualization_seaborn.md

```diff
@@ -0,0 +1,684 @@
+* reading in data
+* manipulating data frames, including calculating new columns
+
+If you are brand new to python (or want a refresher) consider starting with [Intro to python](link) first.
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+
+For the hands-on activities in this module, we will be using an online environment containing a Jupyter notebook. You don't need to install anything or set up an account, but you need a modern web browser like Chrome and a moderately good wifi connection.
+
+Because it can take a few minutes for the environment to be created, we suggest you click the link below to start up the activity. We recommend using right-click to open it in a new tab or window, and then returning here to continue learning while the environment finishes loading. Here is the link:
+
+<a href = "https://mybinder.org/v2/gh/arcus/education_modules/data_viz_seaborn?labpath=data_visualization_in_seaborn%2Fnotebooks%2Fdata_visualization_seaborn.ipynb" target = "_blank"> <img src = "https://mybinder.org/badge_logo.svg"></a> **← Click the "launch binder" button!**
+
+If you have python already installed on your computer and you prefer to work through code examples there, you can <a href="https://raw.githubusercontent.com/arcus/education_modules/main/data_visualization_in_seaborn/notebooks/data_visualization_seaborn.ipynb" download>download the code for this module to run offline</a>.
+
+### Importing seaborn
+
+To use the seaborn library to make visualizations, you need to import it.
+
+By convention, you import it with the shorthand `sns`. This is optional, but we do recommend you do that because when you look at seaborn code online, you'll nearly always see it with that same abbreviation; following that convention will make your code more comparable to examples you see online.
+
+```Python
+import seaborn as sns
+```
+
+### Working through interactive coding examples
+
+
+Hopefully your [binder instance](#lesson-preparation) is done loading now! If not, be patient --- it can take as long as 20 or 30 minutes some times if the files haven't been used recently.
+
+When it is ready, you should see a jupyter notebook in your browser with the code for this module. While you read through this module, we recommend you keep returning back to the binder instance to try running the code for yourself. Even better, try changing the code and see what happens.
+
+<div class = "important">
+Note that binder instances aren't stable. When you close the window or if it idles too long, it may erase all of your work. If you want to save any code or output you come up with while working in binder, you need to copy-paste the code to a new file to save it on your computer.
+</div>
+
+
+### The data
+
+First, we need to load the modules we'll be using:
+
+```python
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+```
+
+<div class = "learnmore">
```

Comment:
Yeah, I think that's a good idea. It will be a change in the css file, though, not this module. So I'm resolving it here for the purposes of this review.